src/analyze/analyze-security.c

```diff
@@ -1775,7 +1781,6 @@ static int assess(const SecurityInfo *info,
 
                 if (details_table) {
                         const char *checkmark, *description, *color = NULL;
```

Comment:
please change `checkmark` to be of type `int`here. Then assign -1 if badness is UINT64_MAX, 0 if cross mark, and 1 if check mark. 

Then add a TABLE_EMPTY cell from it if negative, otherwise TABLE_BOOLEAN_CHECKMARK